.travis.yml

```diff
@@ -0,0 +1,60 @@
+# Copyright 2018 JanusGraph Python Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+language: python
+
+python:
+- '3.6'
+- '3.5'
+- '3.4'
+- '3.5-dev'
+- '3.6-dev'
+
+env:
+  global:
+    - ENV_NAME=tmpenv
+
+install:
+  - pip install virtualenv
+  # Create Virtual env within which all packages will be installed. Default Python version is Travis' Python version
+  - virtualenv "${ENV_NAME}"
+
+before_script:
+  # Install all pre-requisites for building library and docs
+  - bash before-script.sh "${ENV_NAME}"
```

Comment:
`BUILDING.md` doesn't mention this script or the need to set the virtual env. Do I still have to do that before I can build the library or why are you doing it on Travis?
It would be good if Travis would do exactly the same steps I'm supposed to be doing on my machine to build the library / the docs. Otherwise I'm afraid that we will get in a situation where it's working on Travis but not locally or vice versa.

Reply:
I actually broke the the steps part of `build.sh` into travis mentioned steps, i.e. before, during and after steps.
Let me do one thing, in new commit, I'm gonna not break it down as in current travis scrip but make use of single `build.sh` file like mentioned in docs so as to make Docs and Travis same.